docs/getting-started-guides/ubuntu/rancher.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Rancher Integration with Ubuntu Kubernetes
+---
+
+{% capture overview %}
+This repository explains how to deploy Rancher 2.0alpha on Canonical Kubernetes. 
+
+These steps are currently in alpha/testing phase and will most likely change. 
+
+The original documentation for this integration can be found here [https://github.com/CalvinHartwell/canonical-kubernetes-rancher/](https://github.com/CalvinHartwell/canonical-kubernetes-rancher/). 
+
+{% endcapture %}
+{% capture prerequisites %}
+This page assumes you have a working Juju deployed Canonical Kubernetes Cluster. 
+{% endcapture %}
+
+
+{% capture steps %}
+## Implementation
+
+To deploy Rancher, we just need to run the Rancher container workload on-top of Kubernetes. Rancher provides their containers through dockerhub ([https://hub.docker.com/r/rancher/server/tags/](https://hub.docker.com/r/rancher/server/tags/)) and can be downloaded freely from the internet. If you're running your own registry or have an offline deployment, the container should be downloaded and pushed to the private registry.  
+
+## Deploying Rancher
+
+To deploy Rancher, we just need to run the Rancher container workload on-top of Kubernetes. Rancher provides their containers through dockerhub ([https://hub.docker.com/r/rancher/server/tags/](https://hub.docker.com/r/rancher/server/tags/)) and can be downloaded freely from the internet. If you're running your own registry or have an offline deployment, the container should be downloaded and pushed to the private registry.  
+
+### Deploying Rancher with a nodeport
+
+First create a yaml file which will deploy Rancher on kubernetes, save the file as [cdk-rancher-nodeport.yaml](https://raw.githubusercontent.com/CalvinHartwell/canonical-kubernetes-rancher/master/cdk-rancher-nodeport.yaml): 
```

Comment:
wording suggestion; change:
> create a yaml file which will deploy Rancher

to:
> create a yaml file that defines how to deploy Rancher

Also, you probably don't need to reference the YAML both at a remote github site and include it below. If you include it with this PR as an inline yaml, our CI system will validate it, and you can reference it directly using {% include %} to have it display visually as well.

Reply:
Fixed.